recipes/types-s3transfer/meta.yaml

```
@@ -0,0 +1,51 @@
+{% set name = "types-s3transfer" %}
+{% set version = "0.6.0.post1" %}
+{% set lib_version = version.split(".post")[0] %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/types-s3transfer-{{ version }}.tar.gz
+  sha256: 78479647a44c9f94ca4c0d2b90f8da77013daf35485c8786b618136fb7b4501b
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+outputs:
+  - name: types-s3transfer
+    requirements:
+      run:
+        - python >=3.6
+    build:
+      noarch: python
+      script:
+        - python -m pip install . -vv
+    test:
+      script: run_test.py
+      commands:
+        - pip check
```

```suggestion
        - pip check
        - python -c "import importlib; importlib.import_module('s3transfer-stubs')"
```

Is this really the right way to install stub files? I'm not too sure, but it seems strange to have package names with `dashes`.

---

Yes, stub package for `mylib` should be named `mylib-stubs`

---

Fixed, thanks for the suggestion.